doc/languages-frameworks/qt.section.md

```diff
@@ -0,0 +1,113 @@
+
+Qt
+=
+This section describes the differences between Nix expressions for Qt libraries and applications and Nix expressions for other C++ software. Some knowledge of the latter is assumed.
+
+There are primarily two problems which the Qt infrastructure is designed to address: ensuring consistent versioning of all dependencies and finding dependencies at runtime.
+
+**Nix expression for a Qt package (default.nix)**
```

Comment:
Also please preserve the anchors:

```suggestion

## Nix expression for a Qt package (default.nix) {#qt-default-nix}
```

Reply:
Awesome thanks. I was wondering how to get anchors in common markdown, couldn't see it in the spec :).

Reply:
It is a [pandoc’s markdown extension](https://pandoc.org/MANUAL.html#pandocs-markdown) but I think it makes sense to preserve as much of semantics as possible until actual markdown implementation is selected.